frontend/public/components/operator-hub/operator-hub-page.tsx

```diff
@@ -98,7 +98,7 @@ export const OperatorHubPage = withFallback((props: OperatorHubPageProps) => <Re
         isList: true,
         kind: referenceForModel(PackageManifestModel),
         namespace: props.match.params.ns,
-        selector: fromRequirements([{key: 'csc-owner-name', operator: 'DoesNotExist'}]),
+        selector: fromRequirements([{key: 'opsrc-owner-name', operator: 'DoesNotExist'}]),
```

Comment:
This should be `[{key: 'opsrc-owner-name', operator: 'DoesNotExist', key: 'csc-owner-name', operator: 'DoesNotExist'}]`

Reply:
Ah right!

Reply:
Addressed